package.json

```diff
@@ -6,7 +6,6 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node --watch src/server.js",
```

Comment:
Should we prefer `dev` or `start`?

Reply:
https://github.com/monicaleep/member-fe-microsite/pull/11/commits/07546894e5704e63035bc6885da31aef2b49a65e